parquet-cascading/src/main/java/parquet/cascading/ParquetValueScheme.java

```diff
@@ -105,9 +119,17 @@ public ParquetValueScheme(Config<T> config) {
     this.config = config;
   }
 
+  // TODO: Why is there thrift references in this file?
```

Comment:
Filters are defined in terms of the thrift schema and T has to be either a Thrift of Scrooge generated class.
Possibly this should be refactored or renamed.